api/src/migrations/test/1646848536366-public-notice-data.js

```diff
@@ -17,8 +17,13 @@ module.exports = class publicNoticeData1646848536366 {
             , (1108, 1108, 'Suite #1108, 1108th Street, Vancouver BC', 'Monday to Friday 8am to 5pm, Weekends 10am - 3pm', 'P.O. Box 1108 Surrey BC', 'info@co1108ltd.com', 'testdata')
             ;
 
+            -- Records with operation_start_year and operation_end_year
+            UPDATE app_fom.public_notice set operation_start_year = 2025, operation_end_year = 2025 where project_id = 1108;
```

Comment:
This needs to be a separate migration so that it will be run in the test/demo environments, instead of updating the existing migration (that only works in dev when recreating the environment).

Reply:
I thought the test/demo environments also are being recreated for new pr to main branch so I added into existing, I guess was wrong. Will create a new migration for it.

Reply:
Moved to new test migration script.